spec/lib/simp/cli/config/items/action/update_puppet_conf_action_spec.rb

```diff
@@ -7,15 +7,17 @@
 
 require_relative '../spec_helper'
 
+# This test is horribly dangerous since it modifies the actual system.
```

Comment:
Now that you have mocked out the operations, is this statement still true?

Reply:
Nope, it's not. Can remove.